spring-cloud-gcp-docs/src/main/asciidoc/security-iap.adoc

```diff
@@ -0,0 +1,40 @@
+:spring-security-ref: https://docs.spring.io/spring-security/site/docs/current/reference/htmlsingle/
+:spring-security-javadoc: https://docs.spring.io/spring-security/site/docs/current/api/org/springframework/security/
+
+== Google Cloud IAP Authentication
+
+https://cloud.google.com/iap/[Cloud Identity-Aware Proxy (IAP)] provides a security layer over applications deployed to Google Cloud.
+
+The IAP starter uses {spring-security-ref}#oauth2resourceserver[Spring Security OAuth 2.0 Resource Server] functionality to automatically extract user identity from the proxy-injected `x-goog-iap-jwt-assertion` header.
+
+To use IAP authentication, you need to:
```

Comment:
I think you have to use the abbreviation "Cloud IAP" everywhere. We've been dinged for this before when we tried to refer to Cloud Spanner as just "Spanner".
Notice the abbreviation use in [official docs](https://cloud.google.com/iap/).
```suggestion
To use Cloud IAP authentication, you need to:
```